### DISCUSSION TITLE
iOS shortcut

### DISCUSSION THREAD

**Discussion by tomoto3** [deepgram/community#758] [General help] [answered]
### Which Deepgram product feature are you using?

Deepgram API - TTS

### Details

Does someone know how to set "Speech-to-text" API calling by iOS shortcut?? This picture is my best try, but it causes error like this:   {"err_code":"Bad Request","err_msg":"Bad Request: failed to process audio: corrupt or unsupported data","request_id":"395dc89a-c6b7-4fac-88ba-480f16d6d317"}

### If possible, please attach your code or paste it into the text box.

![IMG_0060](https://github.com/deepgram/community/assets/149496794/5d7be335-2155-4eb8-8fb7-2625c4a53782)

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by tomoto3** [comment]
I think below is correct form for a audio file which is provided by URL. However, I want to upload local file using iOS shortcut action.
![Uploading IMG_0064 2.jpeg…]()

**Comment by tomoto3** [comment, marked as answer]
I got solution for Speech-to-Text API calling for iPhone's local file in iOS shortcut. See below picture.


![IMG_0065](https://github.com/deepgram/community/assets/149496794/bd9aefa9-c97b-4404-a9a8-cf372b2742a6)
